.github/workflows/check_code.yml

```
@@ -0,0 +1,48 @@
+name: 🔍 Code Quality Checks
+
+# Срабатывает при пуше на любую ветку, кроме 'main'
+on:
+  push:
+    branches:
+      - '**'
+    branches-ignore:
```

actionlint (1.7.4)

both "branches" and "branches-ignore" filters cannot be used for the same event "push". note: use '!' to negate patterns

(events)